### PR TITLE
 SorrTask514_Clean-up_PyCharm_doc

### PR DESCRIPTION
**Pull request by dhimuk**
Task #514

### CODE REVIEW

**Review by gpsaggese — COMMENTED** [review]
LGTM

File: docs/PyCharm.md [hunk 1]
```diff
@@ -1,4 +1,27 @@
...
+  * [Windows](#windows)
+  * [macOS](#macos)
+  * [Linux](#linux)
+- [Connecting via PyCharm gateway (SSH)](#connecting-via-pycharm-gateway-ssh)
+  * [Connecting via VNC](#connecting-via-vnc)
+- [Configuration](#configuration)
+  * [Reflow](#reflow)
+- [Some recommended plug-ins](#some-recommended-plug-ins)
+
+<!-- tocstop -->
```
Comment (DanilYachmenev) [inline docs/PyCharm.md:25]: Move `# Current situation` header here and make it with 1 # (so it is not a sub-bullet point)
update TOC

File: docs/PyCharm.md [hunk 2]
```diff
@@ -325,9 +348,9 @@ Let's say you are connected via VNC.
 1. Login into the OS.
 2. Run pycharm.sh using terminal (should be there)
 
-    ```
-    > bash /opt/pycharm-community-2021.2.3/bin/pycharm.sh
-    ```
+   ```
+   > bash /opt/pycharm-community-2021.2.3/bin/pycharm.sh
+   ```
```
Comment (DanilYachmenev) [inline docs/PyCharm.md:354]: put all the text in bullet points for consistency
e.g.
```
# How to run our cmamp container directly from PyCharm

- PyCharm allows to run commands directly inside a container
    - See [https://www.jetbrains.com/help/pycharm/using-docker-as-a-remote-interpreter.html (https://www.jetbrains.com/help/pycharm/using-docker-as-a-remote-interpreter.html)
- In fact when we do `i docker_bash` we launch a container and run bash inside it,
but PyCharm can do the same thing
```

do same everywhere in the file

**Review by DanilYachmenev — COMMENTED** [review]
all good, just some formatting and I will merge

File: docs/PyCharm.md
Comment (DanilYachmenev) [inline docs/PyCharm.md:1]: Also rename pls `docs/Tools-PyCharm_figs` -> `docs/PyCharm_figs`

File: docs/PyCharm_figs.md
Comment (DanilYachmenev) [inline docs/PyCharm_figs.md:1]: what I meant is to rename the folder with figures
`docs/Tools-PyCharm_figs` -> `docs/PyCharm_figs`

Comment (dhimuk) [reply docs/PyCharm_figs.md:1]: Oh ok, got it. I was about to say that the images didn't look like they were working

Comment (dhimuk) [reply docs/PyCharm_figs.md:1]: Would you like for the images to be under bullet points too?

File: docs/PyCharm.md (hunk 1 above)
Comment (DanilYachmenev) [reply docs/PyCharm.md:25]: what about this one?

File: docs/PyCharm_figs.md
Comment (DanilYachmenev) [reply docs/PyCharm_figs.md:1]: > Oh ok, got it. I was about to say that the images didn't look like they were working

Yes, I don't see any images displayed in the file preview
this needs to be fixed
try to think of the reason, i'd look to the source code of documentation in the same directory that has images and directory for them (e.g. `docs/Docker.md`)

If nothing works, to save time I'd just reuse the script on the raw docx again and reimplement the discussed changes there with controlling the preview so you do not do changes that break the image display

yes, each corresponding image should be under the bullet point it belongs to
again, see the ref docs

File: docs/PyCharm.md [hunk 3]
```diff
@@ -45,64 +68,55 @@ There are multiple ways to develop on a remote server using PyCharm
...
+- TODO(gp): @Juraj Let's both try this. There are some notes below about it
 
 # How to review a PR inside Pycharm
 
-CTRL + SHIFT + A -> View Pull Request
+- CTRL + SHIFT + A -> View Pull Request
 
-![](docs/Tools-PyCharm_figs/image13.png){width="3.612198162729659in"
-height="4.932292213473316in"}
+- <img src="Pycharm_figs/image13.png" style="width:3.612198162729659in;height:4.932292213473316in" />
```
Comment (DanilYachmenev) [inline docs/PyCharm.md:96]: For some reason images are not being displayed in the Preview
Could you pls double-check and fix it

Comment (zli00185) [reply docs/PyCharm.md:96]: Screenshot 2023-10-12 at 10 25 00 PM

I could see the image here. Could you check it again?

Comment (DanilYachmenev) [reply docs/PyCharm.md:96]: well, you open it from Jupyter which is not the way it is going to be biewed
Use GH preview feature instead

with preview, the problem remains on my side
In the worst case, could you rerun the script pls?

File: docs/PyCharm.md [hunk 4]
```diff
@@ -325,18 +327,16 @@ Let's say you are connected via VNC.
...
 # Configuration
 
 ## Reflow
 
-Set the reflow to reindent
-
-![](docs/Tools-PyCharm_figs/image15.png){width="6.5in"
-height="4.486111111111111in"}
+- Set the reflow to reindent
+  - <img src="Pycharm_figs/image15.png" style="width:6.5in;height:4.486111111111111in" />
```
Comment (DanilYachmenev) [inline docs/PyCharm.md:339]: this one is still broken

**Review by DanilYachmenev — COMMENTED** [review]
1 image to fix and lgtm

File: docs/PyCharm.md (hunk 3 above)
Comment (DanilYachmenev) [inline docs/PyCharm.md:96]: Actually, 1 more
Снимок экрана 2023-10-17 в 01 55 52

**Review by DanilYachmenev — COMMENTED** [review]
lgtm

**Comment by DanilYachmenev** [comment]
@why2000 could you pls move changes from this PR to cmamp?

**Review by why2000 — COMMENTED** [review]
A small issue fixed in PR #594

**Pull request merged**